plugins/copyfilelocation.lua

```diff
@@ -1,15 +1,35 @@
 -- mod-version:3
 local core = require "core"
 local command = require "core.command"
+local treeview = require 'plugins.treeview'
+local menu = treeview.contextmenu
 
-command.add("core.docview", {
-  ["copy-file-location:copy-file-location"] = function(dv)
-    local doc = dv.doc
+command.add("core.docview!", {
+  ["doc:copy-file-location"] = function(view)
```

Comment:
Ideally the command name should keep the name of the plugin.

Reply:
but there's an equivalent for the treeview too, do I just merge them?

Reply:
Eh, they're doing different things, so it makes sense for them to be separate.